web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -80,6 +71,20 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
         >
           Show annotations
         </Checkbox>
+        {alertKinds.map((alertKind, i) => {
+          return (
+            <Checkbox
+              defaultChecked
+              id={`${alertKind}-toggler`}
+              wrapperStyles={{ margin: '0 0 0 15px' }}
```

Comment:
Currently the state texts don't line up nicely vertically with their checkboxes (they seem multiple pixels too low) - could that be improved?

![Screenshot from 2020-01-02 23-32-55](https://user-images.githubusercontent.com/538008/71697650-f8d28500-2db8-11ea-8513-9e6b8a24101b.png)

Reply:
Yes, I also notice that but I decided not to touch it because this is just an idea, not the final solution. Here some more ideas
![Screenshot 2020-01-03 00:48:41](https://user-images.githubusercontent.com/10514821/71698112-eeb58400-2dc2-11ea-866b-c2f3526b0f7c.png)
![Screenshot 2020-01-03 00:55:50](https://user-images.githubusercontent.com/10514821/71698351-e578e700-2dc3-11ea-93dd-5c58f9b25890.png)
![Screenshot 2020-01-03 00:57:27](https://user-images.githubusercontent.com/10514821/71698403-15c08580-2dc4-11ea-80a7-ef2354d30d91.png)